securitycenter/docs/snippets_findings.py

```diff
@@ -0,0 +1,603 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Examples of working with source and findings in Cloud Security Command Center."""
+
+from itertools import chain
+import os
+import pytest
+
+
+@pytest.fixture(scope="module")
+def organization_id():
+    """Get Organization ID from the environment variable """
+    return os.environ["GCLOUD_ORGANIZATION"]
+
+
+@pytest.fixture(scope="module")
+def source_name(organization_id):
+    from google.cloud import securitycenter as securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+    org_name = "organizations/{org_id}".format(org_id=organization_id)
+
+    source = client.create_source(
+        org_name,
+        {
+            "display_name": "Unit test source",
+            "description": "A new custom source that does X",
+        },
+    )
+    return source.name
+
+
+def test_create_source(organization_id):
+    """Create a new findings source. """
+    # [START create_source]
+    from google.cloud import securitycenter as securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+    # organization_id is the numeric ID of the organization. e.g.:
+    # organization_id = "111122222444"
+    org_name = "organizations/{org_id}".format(org_id=organization_id)
+
+    created = client.create_source(
+        org_name,
+        {
+            "display_name": "Customized Display Name",
+            "description": "A new custom source that does X",
+        },
+    )
+    print("Created Source: {}".format(created.name))
+    # [END create_source]
+
+
+def test_get_source(source_name):
+    """Gets an existing source."""
+    # [START get_source]
+    from google.cloud import securitycenter as securitycenter
+    from google.protobuf import field_mask_pb2
+
+    client = securitycenter.SecurityCenterClient()
+
+    # source_name is the resource path for a source that has been
+    # created previously (you can use list_sources to find a specific one).
+    # Its format is:
+    # source_name = "organizations/{organization_id}/sources/{source_id}"
+    # e.g.:
+    # source_name = "organizations/111122222444/sources/1234"
+    source = client.get_source(source_name)
```

Comment:
What if this fails?

Reply:
See above comment, the test fails.  Let me know if this there is something you would like me to assert or demonstrate here.